setup.py

```diff
@@ -22,7 +22,7 @@
         'Cython'
     ],
     install_requires=[
-        'pandas==1.3.0',
+        'pandas>=1.3.0',
```

Comment:
How does `>=` work? Does it mean that only the last digit in the version can change (i.e. from 1.3.0 to 1.3.99, but not to 1.4.0) or all versions above are accepted?

Reply:
I just wanted to test it out this way, had no idea that it'd work lol